apache-couchdb-mixin/dashboards/couchdb-nodes.libsonnet

```diff
@@ -0,0 +1,1268 @@
+      'rate(couchdb_httpd_bulk_requests_total{' + matcher + '}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}"',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Bulk requests',
+  description: 'The number of bulk requests for a node.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'reqps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local responseStatusOverviewPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"2.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}",status="2xx"',
+    ),
+    prometheus.target(
+      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"3.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}",status="3xx"',
+    ),
+    prometheus.target(
+      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"4.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}",status="4xx"',
+    ),
+    prometheus.target(
+      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"5.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}",status="5xx"',
+    ),
+  ],
+  type: 'piechart',
+  title: 'Response status overview',
+  description: 'The responses grouped by HTTP status type (2xx, 3xx, 4xx, 5xx) for a node.',
```

Comment:
Only include values > 0 and consider splitting this in two panels so healthy responses and unhealthy responses aren't hidden by the nature of one being much larger than the other, especially given the linear axis used in this panel